src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public static void doWork() {
+        System.out.println("Bulldozer started its work");
+
+    }
+
+    public static void stopWork() {
+        System.out.println("Bulldozer stopped working ");;
```

Comment:
There is an unnecessary semicolon at the end of the line that should be removed.